configserver/src/main/java/com/yahoo/vespa/config/server/application/ZKTenantApplications.java

```diff
@@ -151,10 +161,10 @@ private void applicationAdded(ApplicationId applicationId) {
         log.log(LogLevel.DEBUG, Tenants.logPre(applicationId) + "Application added: " + applicationId);
     }
 
-    private void removeApplications(PathChildrenCacheEvent.Type eventType) {
+    private void removeApplications() {
         ImmutableSet<ApplicationId> activeApplications = ImmutableSet.copyOf(listApplications());
-        log.log(LogLevel.DEBUG, "Got " + eventType + " event for tenant '" + tenant +
-                "', removing applications except these active applications: " + activeApplications);
+        log.log(LogLevel.INFO, "Removing stale applications for tenant '" + tenant +
```

Comment:
This may become too noisy? How about logging at INFO in removeApplicationsExcept for those applications that are actually removed by this?

Reply:
There is logging in removeApplicationsExcept already, so yes should lower to DEBUG.